src/pathfora.js

```diff
@@ -3593,6 +3607,17 @@
           context.pathfora.closeWidget(widget.id, true);
         }, widget.displayConditions.hideAfter * 1000);
       }
+
+      core.widgetResizeListener(widget, node);
+
+      var resize = window.onresize;
+      window.onresize = function (event) {
+        if (typeof resize === 'function') {
```

Comment:
this is going to always evaluate to true. were you going for a skip the resize if resize was undefined before hitting like 3613?